### ISSUE TITLE
getting warning from three.js on every tick that seems to crash page if console is up

### ISSUE THREAD

**Issue by kylebakerio** [wmurphyrd/aframe-super-hands-component#189]
My project seems to be working in quest, I essentially made the changes described in my 'documenting' issue, however, when I open my project on desktop now, I see this warning from THREE:

```js
THREE.Quaternion: .inverse() has been renamed to invert().
```

on loop, infinite, seemingly at the pace of a requestAnimationFrame() call, and the first time I sat there watching it it eventually crashed my computer entirely which did a hard reset on its own suddenly.

I assume it's only a problem while the console is open, but not sure yet, haven't run oculus in adb bridge with console open since I've added this yet, but if I do and notice it there still, I'll update here... but given the message being logged, I imagine I would see it there if log was open too.

**Comment by kylebakerio** [comment]
I just realized that it's from the physics repo, not here, so I'll open an issue there I guess.
```js

nverse | @ | three.js:37223
-- | -- | --
  | Body.syncFromPhysics | @ | aframe-physics-system.js:17403
  | step | @ | aframe-physics-system.js:17280
  | tick | @ | aframe-physics-system.js:19025
  | tick | @ | a-scene.js:746
  | render | @ | a-scene.js:790
  | bound | @ | bind.js:12


```

**Comment by kylebakerio** [comment]
I followed this wormhole... it's really nothing the physics repo could do--I mean, yes, they can update to be compatible with latest version of A-Frame...

...but A-Frame should block the THREE logs...

...and THREE should make that easy and handle that itself...

...but I just read for hours arguments taking place over years on the THREE repo, and mrdoob basically refuses to do this. As I just posted there on the last one I saw, it's like reading a kafka novel.

Anyways, going to go add a console hack to block THREE messages and let the physics library maintainer know he needs to update so console.warn in tick() doesn't cause implosions, I guess. :/
